<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.highq.test.siteAdmin.SiteAdmin_TC0041" tests="2" time="0.098" ignored="0" failures="0" timestamp="03 Jul 2018 10:25:14 GMT" skipped="0" hostname="HQI-194-RONAKYB" errors="2">
  <testcase name="tearDown" time="0.000" classname="com.highq.test.siteAdmin.SiteAdmin_TC0041">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.highq.base.TestBaseSetup.tearDown(TestBaseSetup.java:380)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- tearDown -->
  <testcase name="TC0041" time="0.098" classname="com.highq.test.siteAdmin.SiteAdmin_TC0041">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:882)
	at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:104)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
	at com.highq.base.TestBaseSetup.areSpinnersLoaded(TestBaseSetup.java:469)
	at com.highq.base.TestBaseSetup.waitAndFindElement(TestBaseSetup.java:222)
	at com.highq.base.TestBaseSetup.findVisibleElement(TestBaseSetup.java:198)
	at com.highq.pageobjects.pages.LoginPageWeb.setEmailId(LoginPageWeb.java:38)
	at com.highq.pageobjects.pages.BannerPageWeb.login(BannerPageWeb.java:1035)
	at com.highq.test.siteAdmin.SiteAdmin_TC0041.scenario2(SiteAdmin_TC0041.java:70)
	at com.highq.test.siteAdmin.SiteAdmin_TC0041.TC0041(SiteAdmin_TC0041.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC0041 -->
</testsuite> <!-- com.highq.test.siteAdmin.SiteAdmin_TC0041 -->
